src/ORM/Table.php

```diff
@@ -460,17 +460,17 @@ public function schema($schema = null)
      * ### Example:
      *
      * ```
-     * protected function _initializeSchema(\Cake\Database\Schema\Table $table) {
+     * protected function _initializeSchema(\Cake\Database\Schema\TableSchema $table) {
      *  $table->columnType('preferences', 'json');
      *  return $table;
      * }
      * ```
      *
-     * @param \Cake\Database\Schema\Table $table The table definition fetched from database.
-     * @return \Cake\Database\Schema\Table the altered schema
+     * @param \Cake\Database\Schema\TableSchema $table The table definition fetched from database.
+     * @return \Cake\Database\Schema\TableSchema the altered schema
      * @api
      */
-    protected function _initializeSchema(Schema $table)
+    protected function _initializeSchema(TableSchema $table)
```

Comment:
@markstory I didn't know that :stuck_out_tongue:
Then #8612 and #8709 might cause strict errors too, if people are using overridden methods having effected typehints in their applications. It would be better to document it when we release them.